bitrepository-client/src/main/java/org/bitrepository/client/conversation/mediator/CollectionBasedConversationMediator.java

```diff
@@ -136,10 +136,10 @@ public void onMessage(Message message, MessageContext messageContext) {
     }
 
     /**
-     * Will clean out obsolete conversations in each run. An obsolete conversation is a conversation which satisfies on
-     * of the following criterias: <ol>
+     * Will clean out obsolete conversations in each run. An obsolete conversation is a conversation which satisfies ony
```

Comment:
"ony"? :)

Reply:
:-) My eyes, I had read "an" and had wanted to change to "any". Corrected to "one" now.